templates/repo/settings/options.tmpl

```diff
@@ -201,6 +200,7 @@
 								</td>
 							</tr>
 						</tbody>
+						{{end}}{{/* end if: mirror is broken or working */}}
```

Comment:
```suggestion
						{{end}}{{/* end if: $modifyBrokenPullMirror */}}
```

Reply:
But there is also an "else if $isWorkingPullMirror". I would like to keep both of them like this.

```suggestion
						{{end}}{{/* end if: $modifyBrokenPullMirror {{else}} $isWorkingPullMirror */}}
```

Reply:
Fine with me, but I prefer just picturing what is in the opening `if`.